common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -356,6 +356,10 @@ def get_data_for_vertical(self, vertical):
                     return (group.name, group.id)
         return (None, None)
 
+    @property
+    def tooltip_content(self):
+        return getattr(self.child, 'tooltip_content', '') if self.child else ''
```

Comment:
I don't think the if/else is required here.  If `self.child` is a false value (presumably we're expecting `None`), the `getattr` will return `''`.  If `self.child` is not defined, either version will raise an `AttributeError`.  The only situation I can think of where the conditional would change the outcome is if `self.child` were a falsy object with a `tooltip_content` attribute.  That doesn't seem like a reasonable case.